pkg/translator/loki/logs_to_loki.go

```diff
@@ -106,3 +114,18 @@ func LogsToLoki(ld plog.Logs) (*logproto.PushRequest, *PushReport) {
 
 	return pr, report
 }
+
+func handleSeverity(log plog.LogRecord) {
+	addHint(log)
```

Comment:
Can just `return` if `log.SeverityNumber() == plog.SeverityNumberUndefined` before adding the hint and attribute?